app/services/onlinetesting/Phase2TestService.scala

```diff
@@ -296,18 +307,19 @@ trait Phase2TestService extends OnlineTestService with Phase2TestConcern with Sc
     }
   }
 
-  //TODO Once the time adjustments ticket has been done then this should be updated to apply the etray adjustment settings.
-  def buildTimeAdjustments(needsAdjustment: Boolean, assessmentId: Int) = if (needsAdjustment) {
-    List(TimeAdjustments(assessmentId, sectionId = 1, absoluteTime = 100))
-  } else {
-    Nil
+  def buildTimeAdjustments(assessmentId: Int, application: OnlineTestApplication) = {
+    if (application.needsAdjustments || application.guaranteedInterview) {
+      List(TimeAdjustments(assessmentId, sectionId = 1, absoluteTime = calculateAbsoluteTimeWithAdjustments(application)))
```

Comment:
It seems there are test with several sections and you can configure every section with different time.